client/src/common/components/Buttons/BaseButton.tsx

```diff
@@ -18,23 +18,28 @@ export type BaseButtonProps = {
   active?: boolean;
 };
 
-const BaseButton = styled(TouchableOpacity)<BaseButtonProps>(
-  ({variant, small, elevated, disabled, active}) => ({
-    backgroundColor: disabled
-      ? COLORS.GREYMEDIUM
-      : active
-      ? COLORS.ACTIVE
-      : variant === 'secondary'
-      ? COLORS.BLACK
-      : variant === 'tertiary'
-      ? COLORS.WHITE
-      : COLORS.PRIMARY,
-    borderRadius: small ? SPACINGS.TWELVE : SPACINGS.SIXTEEN,
-    alignItems: 'center',
-    justifyContent: 'center',
-    flexDirection: 'row',
-    ...(elevated ? SETTINGS.BOXSHADOW : {}),
-  }),
-);
+const BaseButton = styled(TouchableOpacity).attrs({
```

Comment:
@gewfy do you want to help me understand hitSlop? I've tried testing but can't validate the impact of this change. Also should be specific depending on type of button, If I understand it right (which I probably don't)